packages/typespec-ts/test/modularIntegration/lroStardard.spec.ts

```diff
@@ -0,0 +1,314 @@
+import {
+  StandardClient,
+  User
+} from "./generated/lro/standard/generated/src/index.js";
+import { assert } from "chai";
+import { restorePoller } from "./generated/lro/standard/generated/src/restorePollerHelpers.js";
+import { OperationState } from "@azure/core-lro";
+
+describe("LROStandardClient Classical Client", () => {
+  let client: StandardClient;
+
+  beforeEach(() => {
+    client = new StandardClient({
+      allowInsecureConnection: true
+    });
+  });
+
+  describe("createOrReplace", () => {
+    it("just call without await", async () => {
+      client.createOrReplace("madge", {
+        role: "contributor"
+      } as any);
```

Comment:
we should be asserting something here, otherwise there is no much point for the test